variants/pundit/template.rb

```diff
@@ -0,0 +1,46 @@
+source_paths.unshift(File.dirname(__FILE__))
+
+puts "Adding pundit to Gemfile"
+
+gem "pundit"
+
+run "bundle install"
+run "rails g pundit:install"
+run "rails g pundit:policy example"
+
+copy_file "spec/policies/example_policy_spec.rb", force: true
+
+# Configure app/controllers/application_controller.rb
+insert_into_file "app/controllers/application_controller.rb",
+                 after: /^class ApplicationController < ActionController::Base\n/ do
+  <<-RUBY
+    include Pundit
+
+    after_action :verify_authorized, except: %i[index], unless: :current_devise_controller?
```

Comment:
I think it might be better for this `after_action to reference a more generaic method name like `unless :skip_verify_authorized?` because there might be reasons other than devise to skip a policy check. Then `skip_verify_authorized?` could check for devise - this mirrors the pattern you are using in `unless :skip_policy_scoped_controller?`

Reply:
I've changed it to `skip_verify_authorized?`